config/routes.rb

```diff
@@ -1,9 +1,10 @@
 Rails.application.routes.draw do
+  get 'welcome/index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.